main.py

```diff
@@ -1,12 +1,15 @@
 # -*- coding: utf-8 -*-
 from telegram.ext import Updater
-import logging
+import logging, yaml
 
 from module.handlers import handlers
 
 def main() -> None:
-    TOKEN = 'TOKEN'
-    updater= Updater(TOKEN, use_context=True)
+    
+    with open('config/settings.yaml.dist', 'r') as yaml_config:
```

Comment:
we could ignore it for now, but ideally we should have:
- settings.yaml.dist as default conf, but of course there is no TOKEN as default
- settings.yaml the real conf that will be use in production, indeed the file settings.yaml should be in .gitignore, this file is created by the developer to configure this project